Sphere/Sphere.Godot/Nodes/MainNode.cs

```diff
@@ -0,0 +1,150 @@
+using Godot;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Sphere.Common.Interfaces;
+using Sphere.Godot.Configuration.Services;
+using Sphere.Server.Configuration.Options;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Sphere.Godot.Nodes
+{
+    public partial class MainNode : Node
+	{
+        SceneTree tree = new SceneTree();
+
+		public static void Main(string[] args)
+		{
+            
+        }
+
+        public override void _Ready()
```

Comment:
You can mark this async and make all server stuff async as well. It will be an async void though